From 359e35432131c71f181bcfbca20054342448a147 Mon Sep 17 00:00:00 2001
From: Felipe Balbi <balbi@ti.com>
Date: Mon, 14 Feb 2011 19:28:46 +0200
Subject: [PATCH 0297/1092] arm: omap: cbus: add IDs for Retu and Tahvo

Each driver is defining it's own ID. Defining it
on the generic header will allow to pass those
via platform_data.

Signed-off-by: Felipe Balbi <balbi@ti.com>
---
 arch/arm/plat-omap/include/plat/cbus.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/plat-omap/include/plat/cbus.h b/arch/arm/plat-omap/include/plat/cbus.h
index 1f0e8be..e7ace57 100644
--- a/arch/arm/plat-omap/include/plat/cbus.h
+++ b/arch/arm/plat-omap/include/plat/cbus.h
@@ -22,6 +22,9 @@
 #ifndef __PLAT_CBUS_H
 #define __PLAT_CBUS_H
 
+#define CBUS_RETU_DEVICE_ID	0x01
+#define CBUS_TAHVO_DEVICE_ID	0x02
+
 struct cbus_host_platform_data {
 	int	dat_gpio;
 	int	clk_gpio;
-- 
1.7.11.2

